index.js

```diff
@@ -263,10 +264,12 @@ AFRAME.registerComponent('teleport-controls', {
       if (!this.data.cameraRig) {
         var hands = document.querySelectorAll('a-entity[tracked-controls]');
         for (var i = 0; i < hands.length; i++) {
-          var position = hands[i].getAttribute('position');
-          var diff = auxVector3.copy(rigWorldPosition).sub(position);
-          var newPosition = auxVector3.copy(newRigWorldPosition).sub(diff);
```

Comment:
@netpro2k I didn't spot the problem when I just reviewed the code without testing it. As you're using `auxVector3` reference in the `diff` but before that it's being updated as it's used also for copy `newRigWorldPosition`, the `newPosition` is always `(0,0,0)`